code/modules/mob/living/carbon/human/species.dm

```diff
@@ -225,6 +225,7 @@
 	brute_mod = 0.55
 	burn_mod = 0.55
 	unarmed_type = /datum/unarmed_attack/punch/strong
+	flags = IS_WHITELISTED
```

Comment:
i don't think we need a whitelist

Reply:
You're looking at Vox Armalis, which are direct upgrades in damage and defense. The normal Vox aren't whitelisted.

Reply:
i see